pip/_vendor/html5lib/filters/alphabeticalattributes.py

```diff
@@ -5,7 +5,7 @@
 try:
     from collections import OrderedDict
 except ImportError:
-    from pip._vendor.ordereddict import OrderedDict
+    from ordereddict import OrderedDict
```

Comment:
If `_vendor.orderdict` has also been removed, should this be only the `from collections` line now?